release-notes.txt

```diff
@@ -1,3 +1,503 @@
+RIOT-2021.04 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2021.04 release includes:
+
+- netstats can now collect L1/L2 per neighbor statistics
+- sock DTLS now allows using multiple credentials
+- congure - a framework for implementing congestion control algoritms
+- xfa (cross file arrays) - a mechanism to more easily define stuff across 
+  files (e.g., shell commands)
+- many more modules and drivers have Kconfig support
+- the semtech-loramac package now supports sx126x/llcc68 radios
+- 7 new boards, 5 new sensor drivers, 8 packages upgraded
+
+We've also mostly finished our migration from mailing lists to forum.riot-os.org.
+
+375 pull requests, composed of 992 commits, have been merged since the
+last release, and 16 issues have been solved. 56 people contributed with
+code in 103 days. 2564 files have been touched with 191220 (+) insertions and
+91400 deletions (-).
+
+
+Notations used below:
+=====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+Core
+----
++ clist: added clist_is_empty()
++ xfa: intitial support for XFA's (cross file arrays)
+
+System Libraries
+----------------
+* clif: fix attribute parsing
+* shell: allow defining shell commands using XFA
++ dbgpin: module for debugging and profiling
++ ztimer: add ztimer_is_set()
+
+Networking
+----------
+
+* cord: Discovery patch altered to reflect changes in the underlying Resource Directory draft
+* lwip: hook up sam0_eth support
++ congure: framework to implement congestion control and use it in a reusable manner with
+  a number of protocols
++ netstats: L1/L2 per neighbor statistics
++ sock DTLS: allow using multiple credentials
+
+Packages
+--------
+
++ pkg/semtech-loramac: add support for sx126x/llcc68 radios
```

Comment:
The mention to sx126x and llcc68 should go to the "device drivers" section. The change in the loramac package was about using the generic netdev api instead of the sx127x specific API.

Reply:
added them to drivers